proxy-robots-module/src/main/java/orwell/proxy/robot/IPWebcam.java

```diff
@@ -0,0 +1,58 @@
+package orwell.proxy.robot;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.config.IConfigCamera;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class IPWebcam implements ICamera {
+    private final static Logger logback = LoggerFactory.getLogger(IPWebcam.class);
+    private final static String DUMMY_URL = "http://goo.gl/y1QTB7";
+    private URL url;
+
+    public IPWebcam(final URL url) {
+        this.url = url;
```

Comment:
After a lot of internal struggle, I eventually agree to throw a specific exception in the constructor, and also to add some assert on the parameters.